logic/resolvable/Concludable.java

```diff
@@ -182,8 +192,8 @@ public Conjunction conjunction() {
 
     public static class Relation extends Concludable<RelationConstraint> {
 
-        public Relation(final RelationConstraint constraint) {
-            super(ConstraintCopier.copyConstraint(constraint));
+        public Relation(final RelationConstraint constraint, @Nullable IsaConstraint isaConstraint) {
+            super(ConstraintCopier.copyConstraint(constraint), isaConstraint == null ? set() : set(isaConstraint));
```

Comment:
and these constructors will take a Conjunction of the correct shape already i think, then save pointers to the `isa` and `relation` constraints in that conjunction right?

Reply:
Also sounds correct 🤔